### ISSUE TITLE
Interupted NFC read doesn't contiune

### ISSUE THREAD

**Issue by legomannetje** [flipperdevices/flipperzero-firmware#1666]
### Describe the bug.

If you are just out of reach of an NFC card while reading a card (in my case MF classic) and while it's running the dict, it wont contiune after providing the card

### Reproduction

1. Start reading a MF classic card
2. While running the dict attack, remove the card
3. Place the card again. 
4. It won't contiune

### Target

NFC

### Logs

```Text
Don't have logs of it, can try to extract some, but don't know how atm
```

### Anything else?

Running the 0.65.2-RC firmware

**Comment by skotopes** [comment]
Please check latest release

**Comment by Astrrra** [comment]
Closing as fixed by #1868, please re-open if the issue is still there.
